src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
+:root {
+  --color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  display: grid;
+  grid-template: 60px 1fr/ 1fr;
+}
+
+.header {
+  grid-row: 60px;
```

Comment:
The value '60px' for 'grid-row' is incorrect. 'grid-row' should specify the starting and ending row lines, not a fixed height. Consider using 'grid-row: 1;' or similar to specify the row position.